### ISSUE TITLE
Log to stdout with `sbd serve`

### ISSUE THREAD

**Issue by srstsavage** [gadomski/sbd-rs#18]
Would be nice to be able to log directly to stdout when running `sbd serve`, maybe with `--logfile=-` or similar.

When using Docker, a workaround is to log to `/dev/stdout`, making sure to use the exec `CMD` form and not the shell form so that the `sbd serve` process is pid 1 in the container.

```
CMD ["sbd","serve","0.0.0.0:10800","/data","--logfile=/dev/stdout"]
```

**Comment by gadomski** [comment]
Closed by #20 .

**Comment by srstsavage** [comment]
Awesome, thank you!
